components/demo/demo-snippet.js

```diff
@@ -108,6 +110,7 @@ class DemoSnippet extends LitElement {
 				<div class="d2l-demo-snippet-settings">
 					<d2l-switch text="RTL" ?on="${dirAttr === 'rtl'}" @change="${this._handleDirChange}"></d2l-switch><br />
 					<d2l-switch text="Fullscreen" ?on="${this._fullscreen}" @change="${this._handleFullscreenChange}"></d2l-switch><br />
+					<d2l-switch text="French" ?on="${this._isFrench}" @change="${this._handleLanguageChange}"></d2l-switch><br />
```

Comment:
Instead of a switch... what about a dropdown that contains all the languages we support? You could [grab that list from here](https://github.com/BrightspaceUI/intl/blob/main/lib/common.js#L3).

Reply:
The placement of this function doesn't seem quite right to me. The code attached to the switch acts at the document level (good), but there is a switch for each snippet and their state isn't sync'd, which results in some kinda semi-broken behaviour depending on the switch that is subsequently clicked and the current state of the document. See screenshot. 

I've been thinking lately that it would be good to have a page-level set of controls (RTL, language) that operates on the whole document vs snippet-level controls (fullscreen). Maybe part of `d2l-demo-page`. There is an edge case where the RTL switch doesn't work well as coded - happy to go into the details for anyone interested, but I think pulling out the RTL switch is the proper solution (though I have another sol'n in my back pocket). Don't need to address RTL now, but creating that page-level control container may put us on the right path.

![image](https://user-images.githubusercontent.com/9042472/203810717-bbf01db8-c2dc-49f6-b7dc-ebfe82d55c34.png)

Reply:
I like the idea of a page-level thing for RTL and the language even better. Technically if we had a language switcher, choosing Arabic would come with RTL so we might get away with dropping that entirely.

Reply:
I'll put up a draft that attempts a page-level dropdown.

Reply:
HMm..... I'll do my best to get to it.. there hasn't been a lot of time for me to code this sprint. =p

Reply:
Done: https://github.com/BrightspaceUI/core/pull/3015